### ISSUE TITLE
[REQUEST] Module icons for modules currently using duplicates

### ISSUE THREAD

**Issue by ebsmarch** [Liam-Harrison/Celeritas#304]

**Comment by MistaMishMash** [comment]
Task Estimation (time): 2.5 + 1 hours
Does this issue address a risk? not really, it was caused by unrecorded tech debit though this has been addressed in the risk log.

Risks: not really. I still have some things to learn with Figma, so might take me a little longer as I get familiar with some more advanced features.
Risk Resolution: I'll add a little extra time to my estimation (1 hour)

**Comment by MistaMishMash** [comment]
Note: identified duplicate module icons are Vampiric, Stun and Increased Damage Over Distance.

**Comment by MistaMishMash** [comment]
Actual time taken: 3 hours 15 minutes

Testing procedure: Sent the icons to the team for feedback.
Test Pass Condition(s):  Team is happy with the icons

Risks: As discussed earlier. The extra time was required, to fix some issues I had with grey strokes appearing on the exported PNG. I have found a workaround, though the process did cause this task to take longer, so it was good I added extra time to my estimation.
Risk Resolution: The extra time on my estimation was handy! Worked around with the image tracer plugin for Figma.
